apps/web/src/components/SearchRequestTable/SearchRequestTable.tsx

```diff
@@ -87,8 +90,7 @@ export const SearchRequestTable = ({
           description:
             "Title displayed on the search request table requested date column.",
         }),
-        accessor: "requestedDate",
-        Cell: ({ value }) => dateAccessor(value, intl),
+        accessor: ({ requestedDate }) => dateAccessor(requestedDate, intl),
       },
```

Comment:
I think it's sorting the date alphabetically.
![image](https://user-images.githubusercontent.com/8978655/217616348-8b17f69e-da90-45f4-a7b5-999cd896f8a5.png)

Reply:
wouldn't ascending (upwards arrow) be latest to to oldest, in which case April 2024 is newer than Feb 2023

Reply:
I think "arrow up" is smallest at the top.   For example, classification salaries:
![image](https://user-images.githubusercontent.com/8978655/217623632-307991ef-8e23-4dfa-9806-6ee3e4450ef6.png)

Reply:
:+1: